### DISCUSSION TITLE
Bots sharing viruses in issues as a help message

### DISCUSSION THREAD

**Discussion by FawadAbbas12** [community/community#139085] [Projects and Issues]
### Select Topic Area

General

### Body

Hey everyone,

I wanted to share a strange experience that happened to me today. I created a new issue on this [repo](https://github.com/RPL-CS-UCL/DiPPeR/issues/1) — really impressive work in its field, by the way.

Not even a minute later, I received a reply from [albpedmul](https://github.com/albpedmul) who was listed as a contributor. Everything seemed legitimate at first. However, the user no longer exists on GitHub as of now.

In the reply, they posted a link to a mega thread that contained a zip file with two .dll files and an .exe file (which I later confirmed to be a virus). Thankfully, I was using Linux, so I dodged that bullet.

What was really strange was that after I mentioned that the files didn't work for me because I was on Linux, the user's reply, account, and even their contributor status on the repo were all mysteriously deleted.

I’ve heard about unauthorized commit exploits, but this was my first time encountering someone actively distributing malware through a GitHub issue.

Just a heads-up for anyone else who might come across suspicious links or users — stay cautious!

**Comment by geoffturk** [comment]
This just happened to me as well here:
https://github.com/MurmurationsNetwork/MurmurationsServices/issues/824

The bot sent its reply within seconds of me submitting the issue to the repo. I have opened a ticket about bot abuse with GH Support.

**Reply by jahagirdar** [reply]
Came here because the same thing happened with me!
![image](https://github.com/user-attachments/assets/3641e196-b033-4217-96ac-464b46479b2f)

**Comment by LiteBrite82** [comment]
Thanks for the reports @FawadAbbas12, @jahagirdar, @geoffturk !!!
